slices/vim-common.yaml

```diff
@@ -0,0 +1,15 @@
+package: vim-common
+
+essential:
+  - vim-common_copyright
+
+slices:
+  configs:
+    contents:
+      /etc/vim/vimrc:
+      /usr/bin/helpztags:
+      /usr/lib/mime/packages/vim-common:
```

Comment:
I think `/usr/bin/helpztags` and `/usr/lib/mime/packages/vim-common` can be omitted.